ClinGen/clincoded
# Connects to #1515. Connects to #1516. Migrate disease to MONDO IDs.

**Steps to test #1515:**
1. Go to ^/create-gene-disease/ and click on the "Disease+" button.
2. Expect to see the disease modal being shown.
3. In the disease modal, enter **MONDO:0000218** in the text field and click the "Retrieve from OLS" button.
4. Expect to see disease _"Preimplantation embryonic lethality"_ being shown as a service api response in the disease modal.
5. Now change the **MONDO:0000218** ID to **OMIM:100800** in the text field and click the "Retrieve from OLS" button again.
6. Expect to see an error message "Enter a valid MONDO ID" below the text field.

**Steps to test #1516:**
1. Go to the disease collection page.
2. Expect to see the various Orphanet, DOID, OMIM and NCIT being migrated to MONDO IDs, along with their potentially updated terms and definitions as well. For example, **DOID:13636** should be expected to be migrated/mapped to **MONDO:0019391**. And **OMIM:100800** should be expected to be migrated/mapped to **MONDO:0007037**.
